packages/core/admin/admin/src/content-manager/utils/index.js

```diff
@@ -1,3 +1,2 @@
 export { default as arrayMoveItem } from './arrayMoveItem';
 export { default as checkIfAttributeIsDisplayable } from './checkIfAttributeIsDisplayable';
```

Comment:
Still need this file ? Don't we need to convert to TS as well ?

Reply:
I've been converting them when they're used eventually removing them all. I think when I do the list manager next ill be able to remove it.

I've found converting page by page using the data as a source of truth the quickest way to convert without refactoring loads for each edge case I find lol